server/routerlicious/package.json

```diff
@@ -30,7 +30,7 @@
     "docker-build": "docker-compose build",
     "docker-clean": "docker-compose down --rmi local -v",
     "format": "lerna run format --parallel --stream",
-    "preinstall": "node ./scripts/only-pnpm.cjs",
```

Comment:
why this change? Starting paths with ./ when they are relative to the working directory is good practice as it avoids several possible issues (like accidently starting the path at the root or somewhere in the search path).

Did this cause an issue? If so, note this change in the PR description since it can easilly hid with all the other file changes in this PR

Reply:
Yes it caused this issue: file failed policy check: server/routerlicious/pnpm-lock.yamlInvalid pnpm enforcement preinstall script "node ./scripts/only-pnpm.cjs" in package.json
I checked our client repository and saw that it does not had this issue because there was no "./" in there.

Reply:
Added